### PR TITLE
OKTA-394833 - Okta OIE Sign Up page Tests

### PR DESCRIPTION
**Pull request by vigneshnarasimankumar-okta**
## Description:
- **What's changed?** 
- Adding selenium tests for verifying basic components rendering like header buttons, links, sign in and sign up flow of the developer oie page
- **Is this PR related to a Monolith release?** 

### Resolves:

* [OKTA-394833](https://oktainc.atlassian.net/browse/OKTA-394833)

### CODE REVIEW

File: tests/selenium/framework/page-objects/SignUpPage.js [hunk 1]
```diff
@@ -75,6 +81,12 @@ class SignUpPage extends BasePage {
     this.waitForPresence(pageLoadElement);
   }
 
+  navigateToOie() {
+    const pageLoadElement = SignUpPage.getPageLoadElement();
```
Comment (priyakathirvel-okta) [inline tests/selenium/framework/page-objects/SignUpPage.js:85]: can we not call getPageLoadElement directly?

**Comment by priyakathirvel-okta** [comment]
can you maybe attach a screenshot of what we are testing to the PR? I assume you have already checked there are end to end selenium test cases

File: tests/selenium/framework/page-objects/SignUpPage.js (hunk 1 above)
Comment (vigneshnarasimankumar-okta) [reply tests/selenium/framework/page-objects/SignUpPage.js:85]: requires refactoring of the whole class. will keep this for future work

**Review by priyakathirvel-okta — APPROVED** [review]
LGTM. Please follow up on the tests that were removed due to flakiness in this PR

**Pull request merged**